README.adoc

```diff
@@ -56,7 +50,12 @@ To use an existing Yocto build directory:
 [source,console]
 source ./setup-environment build
 
-To start a simple image build:
+If using a fresh docker image, your locale might not be set to en_US.UTF8.
```

Comment:
Who says the user will use docker? Maybe add a separate info point about docker?